diff -u a/oneclient.spec b/oneclient.spec
--- a/oneclient.spec	2016-06-23 03:26:28.160070594 +0200
+++ b/oneclient.spec	2016-06-23 03:26:34.263403929 +0200
@@ -35,6 +35,11 @@
 BuildRequires: swift-sdk-cpp
 BuildRequires: tbb-devel >= 4.3~
 
+BuildRequires: libstdc++-static >= 5.0.0
+BuildRequires: boost-static >= 1.58.0
+BuildRequires: protobuf-static >= 2.6.0
+BuildRequires: glusterfs-api-devel = 3.8.4
+
 %description
 oneclient is a software based on FUSE (Filesystem in Userspace) that allows mounting onedata filesystem on Linux systems.
 
@@ -43,7 +47,8 @@
 
 
 %build
-%cmake . -DCMAKE_BUILD_TYPE=Release -DBUILD_INTEGRATION_TESTS=Off -DBUILD_SHARED_LIBS=Off
+%cmake . -DCMAKE_BUILD_TYPE=Release -DBUILD_INTEGRATION_TESTS=Off -DBUILD_SHARED_LIBS=Off \
+         -DSTATIC_LIBSTDCPP=On -DSTATIC_BOOST=On -DSTATIC_PROTOBUF=On
 make %{?_smp_mflags} oneclient
 
 
